openmetadata-ui/src/main/resources/ui/package.json

```diff
@@ -51,6 +51,7 @@
     "i18next": "^21.10.0",
     "i18next-browser-languagedetector": "^6.1.6",
     "ieee754": "^1.2.1",
+    "immutability-helper": "^3.1.1",
```

Comment:
Are we using this package anywhere?

Reply:
@Sachin-chaurasiya no, we are not removing it.